Jellyfin.Api/BaseJellyfinApiController.cs

```diff
@@ -1,3 +1,4 @@
+using System.Net.Mime;
```

Comment:
@Bond-009 What is the order of imports?

Reply:
I'm fairly confident System imports first is correct. When they are switched, Style Cop Analyzer throws an error.

Reply:
Correct, `System`first and after that alphabetical order